lxd/storage/backend_lxd.go

```diff
@@ -2280,6 +2283,13 @@ func (b *lxdBackend) CreateCustomVolumeFromCopy(projectName string, volName stri
 		desc = srcVolRow.Description
 	}
 
+	// Get the source volume's content type.
+	contentType := drivers.ContentTypeFS
+
+	if srcVolRow.ContentType == "block" {
```

Comment:
@monstermunchkin it would be good to define these DB content type values as constants in the db package so we can reference them here, like the volume types.

Reply:
@monstermunchkin or use a conversion function as @stgraber suggested above, but either way having constants define for use would be good.